docs/cookbook/getTimeZoneObjectFromIanaName.mjs

```diff
@@ -1,29 +1,6 @@
-import assert from 'assert';
+// Construct a Temporal.TimeZone from an IANA name:
+const tz = Temporal.TimeZone.from('Europe/London');
 
-/**
- *
- * @param ianaTimeZoneName {string} TimeZone String
- * @returns {Temporal.Timezone}
- * @example getTimeZoneObjectFromIanaName("Europe/London")
- */
-function getTimeZoneObjectFromIanaName(ianaTimeZoneName) {
-  return new Temporal.TimeZone(ianaTimeZoneName);
-}
-
-const tz = getTimeZoneObjectFromIanaName('Europe/London');
-// Need to cast the timezone back to an IANA string
-assert.equal(tz.toString(), 'Europe/London');
-
-// TimeZone Methods
-
-// Need to get an offset for an Absolute point in time (DST)
-const tzOffset = tz.getOffsetFor(Temporal.Absolute.from('2020-08-06T15:00Z'));
-assert.equal(tzOffset, '+01:00');
-
-// Same as above but during but non-DST
-const tzOffset2 = tz.getOffsetFor(Temporal.Absolute.from('2020-11-06T01:00Z'));
-assert.equal(tzOffset2, '+00:00');
-
-// Get a dateTime object using the timezone as an offset
-const dateTimeOffset = tz.getDateTimeFor(Temporal.Absolute.from('2020-08-06T15:00Z'));
-assert.equal(dateTimeOffset.toString(), '2020-08-06T16:00');
+// Cast the timezone back to an IANA name, two ways:
+tz.toString(); // Europe/London
+tz.name; // Europe/London
```

Comment:
Why not assert those?

Reply:
Readability, mostly... I figured these are tested elsewhere.

What I would _really_ like is to autogenerate assertions out of these comments when running these files, because I think the assertions decrease the readability of all the examples, but I don't know if I should spend a lot of time on that.